alfresco-transform-tika/alfresco-transform-tika/src/main/java/org/alfresco/transformer/metadataExtractors/IPTCMetadataExtractor.java

```diff
@@ -65,9 +72,87 @@ protected Parser getParser() {
      */
     @Override
     protected Map<String, Serializable> extractSpecific(Metadata metadata, Map<String, Serializable> properties,
-            Map<String, String> headers) {
-
+            Map<String, String> headers) 
+    {
         properties = new TikaAutoMetadataExtractor().extractSpecific(metadata, properties, headers);
+        ExifToolParser etParser = (ExifToolParser)this.getParser();
+        if (etParser.getSeparator()!=null)
+        {

```

Comment:
Trivial: code formatting for opening curl brace - use single-style within class / method (& ideally across project codebase)

Reply:
Thanks, updated.